src/cobbler_tftp/settings/schemas/v1_0.py

```diff
@@ -0,0 +1,68 @@
+"""
+Configuration for version 1.0
```

Comment:
```suggestion
Configuration schema 1.0
```

We wanted to decouple the settings file version from the application version. Let's reflect that in the docstring please.